lib/archethic/p2p/message.ex

```diff
@@ -150,6 +152,28 @@ defmodule Archethic.P2P.Message do
     |> List.last()
   end
 
+  @doc """
+  Return timeout depending of message type
+  """
+  @spec get_timeout(__MODULE__.t()) :: non_neg_integer()
+  def get_timeout(message) do
+    full_size_message = [GetTransaction, GetLastTransaction]
+
+    if message.__struct__ in full_size_message do
+      get_max_timeout()
+    else
+      3_000
+    end
+  end
+
+  @doc """
+  Return the maximum timeout for a full sized transaction
+  """
+  @spec get_max_timeout() :: non_neg_integer()
+  def get_max_timeout() do
+    trunc(@content_max_size / @floor_upload_speed * 1_000)
```

Comment:
I think it means  trunc(
(@content_max_size / (@floor_upload_speed * 1_000))
)

Reply:
No it's `trunc((@content_max_size / @floor_upload_speed) * 1_000)`
but formater remove parenthesis as they are not needed in this case